### ISSUE TITLE
Poll

### ISSUE THREAD

**Issue by Kai-Holmes** [burnacid/MyBB-Form-Creator#57]
Would it be possible to set a poll for staff to vote on and to have it left open for a predetermined amount of time, it just needs to be a yes or no poll.

**Comment by burnacid** [comment]
I'm not seeing how this is a option for the form creator. It's just creating a poll on a thread? That is default within MyBB

**Comment by Kai-Holmes** [comment]
I understand the confusion, it was about whether it would be possible to have each thread made by the form creator contain a poll by default.

**Comment by burnacid** [comment]
Well yes but it's not something I would be focusing on.

**Comment by Walter-Correa** [comment]
Yes, this will be nice! Create a poll when you set to create a thread in a forum or post.
